src/Conn.h

```diff
@@ -352,8 +352,8 @@ class Connection final : public Obj {
 	TransportProto proto;
 	uint32_t orig_flow_label, resp_flow_label;	// most recent IPv6 flow labels
 	uint32_t vlan, inner_vlan;	// VLAN this connection traverses, if available
-	u_char orig_l2_addr[Packet::l2_addr_len];	// Link-layer originator address, if available
-	u_char resp_l2_addr[Packet::l2_addr_len];	// Link-layer responder address, if available
+	u_char orig_l2_addr[Packet::L2_ADDR_LEN];	// Link-layer originator address, if available
```

Comment:
I guess that should work. Should that string store the raw data or a formatted representation?

Reply:
Hmm .. I think I take that back, at least for now. It's a change that would propgate through the Connection code. I do think that eventually using the formatted representation as a string here would makes sense (as formatting is the only thing we do with this, and it would make protocol-independent), but ok to leave that for later.